docs/sources/project/find-an-issue.md

```diff
@@ -166,7 +166,7 @@ To sync your repository:
 
 5. Fetch all the changes from the `upstream/master` branch.
 
-        $ git fetch upstream/master
```

Comment:
@moxiegirl this has already been fixed, are you sure you rebased against master?